packages/react/src/components/SidePanel/_side-panel.scss

```diff
@@ -154,3 +153,214 @@ $border-width: 1px;
   z-index: 1;
   box-shadow: 0 0 6px rgba(0, 0, 0, 0.2);
 }
```

Comment:
Probably need to remove all previous css code.

Reply:
Workin on the updates.

Reply:
Same as above. These are angular styles